support/kernel-config/configuration.nix

```
@@ -168,7 +168,8 @@ in
       # Devices using serial I/O; AT keyboard, PS/2 mouse, etc...
       # Option no since some HID devices may `select` it.
       SERIO = if isx86 then yes else (option no);
-      USB_ONBOARD_HUB = whenAtLeast "6.0" yes;
+      USB_ONBOARD_HUB = whenBetween "6.0" "6.9" yes;
+      USB_ONBOARD_DEV = whenAtLeast "6.11" yes;
```

```suggestion
      USB_ONBOARD_DEV = whenAtLeast "6.10" yes;
```
Based on https://cateee.net/lkddb/web-lkddb/USB_ONBOARD_DEV.html, I think this should be 6.10.